docs/setup/install/ingress-setup.md

```diff
@@ -115,6 +112,47 @@ spec:
               number: 80
         path: /grafana
         pathType: ImplementationSpecific  
+```
+
+## Enable HTTPS For Devtron
+
+Once ingress setup for devtron is done and you want to run Devtron over `https`, you need to add different annotations for different ingress controllers and load balancers.
+
+### 1. Nginx Ingress Controller
+
+In case of `nginx ingress controller`, add following annotations under `service.annotations` under nginx ingress controller to run devtron over `https`.
+
+```bash
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-backend-protocol: "http"
+    service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: "<acm-arn-here>"
```

Comment:
This only for aws, what about other cloud providers?

Reply:
Added for digital ocean.

Reply:
Looking for other clouds also.